src/AlgebraicGeometry/Schemes/AffineVariety/Objects/Methods.jl

```diff
@@ -0,0 +1,12 @@
+########################################################
+# (1) Display
+########################################################
+
+function Base.show(io::IO, X::AffineVariety)
+  d = dim(ambient_space(X))
+  println(io, "Affine variety in $ambient_space(X)")
+  println(io, "defined by")
+  println(io, modulus(OO(X)))
```

Comment:
I would expect the first two lines to be in the same line, but this depends on taste....